linkerd/docs/client_tls.md

```diff
@@ -87,3 +87,44 @@ routers:
           certPath: /certificates/cert.pem
           keyPath: /certificates/key.pem
 ```
+
+### Client TLS and transformers
+
+When using any kind of transformer with client TLS using `io.l5d.static`, ensure that the client
+TLS's `prefix` field starts with at least the first transformer that is applied to a client name in 
+the dtab. This will ensure that TLS is initiated between the Linkerd client and any downstream 
+service receiving the intended TLS traffic. For example given a configuration:
+
+```yaml
+routers:
+- protocol: http
+  interpreter:
+      kind: default
+      transformers:
+      - kind: io.l5d.port
+        port: 4141
+  dtab: | 
+    /svc => /#/io.l5d.fs;
+  client:
+    kind: io.l5d.static
+    configs:
+    - prefix: /#/io.l5d.fs/{service}
+      tls:
+        commonName: "{service}.linkerd.io"
+        trustCerts:
+        - /certificates/cacert.pem
+        clientAuth:
+          certPath: /certificates/cert.pem
+          keyPath: /certificates/key.pem
+```
+
+The full client name generated by Linkerd for service `hello` would look like 
+`%/io.l5d.port/#/io.l5d.fs/4141/hello`. Therefore to set up TLS for the service `hello`, the prefix 
+should be `%/io.l5d.port/#/io.l5d.fs/4141`
+
+When multiple transformers are used for a Linkerd instance, it may be difficult to determine the 
```

Comment:
I'd remove the bit about multiple transformers... it can be difficult even if there is a single transformer.